Clockwise.Redis/CircuitBreakerBroker.cs

```diff
@@ -102,11 +91,11 @@ private static string GetKey<T>()
             {
```

Comment:
Shouldn't this be calling `GetPartition<T>` instead of duplicating the dictionary `GetOrAdd` call?